### PR TITLE
Tweak language in the initial telegraf registration for v2.0 alpha

### PR DESCRIPTION
**Pull request by sebito91**
_Briefly describe your proposed changes:_

Update the language for the portion that kicks off telegraf after initial registration of the endpoint.

  - [x] Rebased/mergeable
  - [x] Tests pass
  - [x] http/swagger.yml updated (if modified Go structs or API)
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/dataLoaders/components/verifyStep/TelegrafInstructions.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ class TelegrafInstructions extends PureComponent<Props> {
   public render() {
     const {notify, authToken, configID} = this.props
     const exportToken = `export INFLUX_TOKEN=${authToken || ''}`
-    const configScript = `telegraf -config http://localhost:9999/api/v2/telegrafs/${configID ||
+    const configScript = `telegraf --config http://localhost:9999/api/v2/telegrafs/${configID ||
```
Comment (bthesorceror) [inline ui/src/dataLoaders/components/verifyStep/TelegrafInstructions.tsx]: First of all, Awesome work but as telegraf is a GoLang project it does follow the single dash style, double dashes will not work for this.

File: ui/src/dataLoaders/components/verifyStep/TelegrafInstructions.tsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ class TelegrafInstructions extends PureComponent<Props> {
         <CopyText copyText={exportToken} notify={notify} />
         <h6>3. Start Telegraf</h6>
         <p>
-          Finally, you can run the following command the start Telegraf agent
+          Finally, you can run the following command to start the Telegraf agent
```
Comment (bthesorceror) [inline ui/src/dataLoaders/components/verifyStep/TelegrafInstructions.tsx]: 🍰

File: ui/src/dataLoaders/components/verifyStep/__snapshots__/TelegrafInstructions.test.tsx.snap [hunk 3]
```diff
@@ -33,10 +33,10 @@ exports[`TelegrafInstructions matches snapshot 1`] = `
     3. Start Telegraf
   </h6>
   <p>
-    Finally, you can run the following command the start Telegraf agent running on your machine.
+    Finally, you can run the following command to start the Telegraf agent running on your machine.
   </p>
   <CopyText
-    copyText="telegraf -config http://localhost:9999/api/v2/telegrafs/"
+    copyText="telegraf --config http://localhost:9999/api/v2/telegrafs/"
```
Comment (bthesorceror) [inline ui/src/dataLoaders/components/verifyStep/__snapshots__/TelegrafInstructions.test.tsx.snap]: Same as above about the dashes.

File: ui/src/dataLoaders/components/verifyStep/TelegrafInstructions.tsx (hunk 1 above)
Comment (sebito91) [reply ui/src/dataLoaders/components/verifyStep/TelegrafInstructions.tsx]: This was an early golang `feature` that limited flags to single-dash options but modern implementations of the language now support both. It's usually user-preference, but in this case we should align with `telegraf` itself...

```
[sborza@icebox]:/var/lib/influxdb:$ telegraf --version
Telegraf 1.9.4 (git: HEAD 4da8d0a4)
[sborza@icebox]:/var/lib/influxdb:$ telegraf --help
Telegraf, The plugin-driven server agent for collecting and reporting metrics.

Usage:

  telegraf [commands|flags]

The commands & flags are:

  config              print out full sample configuration to stdout
  version             print the version to stdout

  --aggregator-filter <filter>   filter the aggregators to enable, separator is :
  --config <file>                configuration file to load
  --config-directory <directory> directory containing additional *.conf files
  --debug                        turn on debug logging
  --input-filter <filter>        filter the inputs to enable, separator is :
  --input-list                   print available input plugins.
  --output-filter <filter>       filter the outputs to enable, separator is :
  --output-list                  print available output plugins.
  --pidfile <file>               file to write our pid to
  --pprof-addr <address>         pprof address to listen on, don't activate pprof if empty
  --processor-filter <filter>    filter the processors to enable, separator is :
  --quiet                        run in quiet mode
  --sample-config                print out full sample configuration
  --test                         gather metrics, print them out, and exit;
                                 processors, aggregators, and outputs are not run
  --usage <plugin>               print usage for a plugin, ie, 'telegraf --usage mysql'
  --version                      display the version and exit

Examples:

  # generate a telegraf config file:
  telegraf config > telegraf.conf

  # generate config with only cpu input & influxdb output plugins defined
  telegraf --input-filter cpu --output-filter influxdb config

  # run a single telegraf collection, outputing metrics to stdout
  telegraf --config telegraf.conf --test

  # run telegraf with all plugins defined in config file
  telegraf --config telegraf.conf

  # run telegraf, enabling the cpu & memory input, and influxdb output plugins
  telegraf --config telegraf.conf --input-filter cpu:mem --output-filter influxdb

  # run telegraf with pprof
  telegraf --config telegraf.conf --pprof-addr localhost:6060

```

**Comment by goller** [comment]
Hi @sebito91 , if you would do one quick conflict resolution, I'll merge straight away.

**Comment by sebito91** [comment]
@goller done! Lemme know if there's anything else you need.

**Comment by goller** [comment]
Thanks!

**Pull request merged**